### DISCUSSION TITLE
Any ideas on the upcoming 'production'? 🏭

### DISCUSSION THREAD

**Discussion by Im-Boss-13** [DawnBee/college-medicine-app#117] [Ideas]
Should we make a 'production' branch to encapsulate newly added feature only to 'development' and 'main' branch before reflecting it to the live build?

**Comment by DawnBee** [comment]
> Should we make a 'production' branch to encapsulate newly added feature only to 'development' and 'main' branch before reflecting it to the live build?

That seems to be the best idea here!

**Reply by Im-Boss-13** [reply]
Agreed! even though the app is relatively small, it is always important to follow industry standards! @DawnBee
